services/movie.py

```diff
@@ -0,0 +1,41 @@
+from django.db.models import QuerySet
+
+from db.models import Movie
+
+
+def get_movies(genres_ids: list = None, actors_ids: list = None) -> QuerySet:
+
+    queryset = Movie.objects.all()
+
+    if genres_ids:
+        queryset = queryset.filter(genres__id__in=genres_ids)
+
+    if genres_ids and actors_ids:
+        queryset = queryset.filter(genres__id__in=genres_ids, actors__id__in=actors_ids)
```

Comment:
The condition `if genres_ids and actors_ids:` is redundant because the subsequent `if actors_ids:` will override the filter applied here. Consider removing this condition to avoid unnecessary filtering.